src/Symfony/Bridge/Doctrine/Form/ChoiceList/IdReader.php

```diff
@@ -84,12 +84,12 @@ public function isIntId(): bool
      *
      * This method assumes that the object has a single-column ID.
      *
-     * @return mixed The ID value
+     * @return string The ID value
      */
     public function getIdValue(object $object = null)
     {
         if (!$object) {
-            return null;
+            return '';
```

Comment:
no a BC break because `@internal` - but there is a design smell to me: the constructor of `DoctrineChoiceLoader` requires an `IdReader`, which transitively means it's not "supported" to instantiate it... not directly related to this PR thought.
Which still leads to: is this change worth it? can we do without?